app/views/manage_assignments/coverages/_coverage.html.erb

```diff
@@ -13,6 +13,9 @@
     </h2>
     <p class="class-card-heading-outcomes">
       <%= t(".matched_outcomes", count: coverage.outcome_coverages.size) %>
+    <p>
```

Comment:
This is a good point. @mborsare, What do you think? I lean having a link to Add a syllabus only if there currently is not one. Else, there is a link to the attachment.